tasks/install_tool_list.yml

```diff
@@ -9,6 +9,6 @@
   failed_when: ('Error installing' in install_result.stderr) or ('Missing required' in install_result.stderr) or (install_result.rc != 0)
   ignore_errors: "{{ galaxy_tools_ignore_errors }}"
   notify:
-    - restart galaxy
+    - ansible-galaxy-tools restart galaxy
```

Comment:
@nuwang @nuwang I don't clearly see why a notification is required when tools are installed (given that creating user and deleting user tasks are notified to the handler), but I am almost sure that THIS notification is not properly placed. Could not it be placed in tools.yml or even in main.yml ?

In any case this PR is crashing GalaxyKickStarts and it has something to do with the ansible-galaxy-tools restart galaxy handler.

Reply:
@drosofff Can you provide some more information/error trace of where GalaxyKickStart is failing? The reason that prompted this fix was because the actual handler that was getting called was not the one defined in this role, but a handler defined in a previously included role, which was masking the correct handler. So all this PR does is to make that name unique.  Since a handler will only run once, multiple or even redundant notifications shouldn't cause a problem and therefore, an error that cascades like this seems like an indication that there's a deeper design issue at hand.